content/blog/2022-07-26-iterative-studio-model-registry.md

```diff
@@ -0,0 +1,185 @@
+history, and so on. But this is highly error-prone and will probably get
+out-of-date very quickly. Or maybe you upload all your models to a cloud bucket
+and “attach” text reports to them. Not very maintainable or searchable either.
+We’ve even seen people use sticky notes, or better yet, rely on their memory 😀
+
+Some of the more organized folks use Model Registries - tools created
+specifically to organize models into a central, searchable repository. While
+this is definitely better than using random files or sticky notes, one major
+problem persists: the data science and machine learning team members work
+completely isolated from the software development and DevOps team members. This
+makes collaboration far more time consuming than it should be.
+
+![Teams can work in disconnected silos](/uploads/images/2022-07-26/disconnected-silos.png)
+
+Some even implement in-house systems, and maybe you are also planning to do so.
+But these can get expensive to develop and maintain.
+
+This is a common problem, and deserves a robust, intuitive and scalable
+solution.
+
+**We built the Iterative Studio Model Registry to solve these problems.**
+
+Iterative Studio Model Registry enables ML teams to collaborate on models by
+providing model organization, discovery, versioning, lineage (tracing the origin
+of the model), and the ability to manage deployment statuses such as,
+development, staging, and production across multiple projects.
+
+## Utilize your existing Git infrastructure
+
+Iterative Studio Model Registry is built on top of Git, which means:
+
+- You can reuse your existing Git infrastructure to manage ML models together
+  with code, data, experiment pipelines, and deployment statuses.
+- You can use GitOps for model deployment, and trigger model deployment from
+  Iterative Studio, which you can also use to run your ML experiments.
+- Your DS/ML folks and Software/DevOps folks can work together more easily,
+  because they utilize the same tools and infrastructure.
+
+## Open MLOps
+
+A core philosophy at Iterative is open MLOps - we build tools that work with
+your infrastructure. Our toolstack is modular, so you can build your model
+registry on top of your existing cloud and DevOps infrastructure and avoid
+vendor lock-in.
+
+Under the hood, Iterative Studio Model Registry uses Iterative’s open-source
+Git-based tools [GTO] and [MLEM].
+
+- [GTO] enables [semantic versioning][semver] and stage transitions of artifacts
+  using metadata files and Git tags.
+- [MLEM] saves ML models and extracts model metadata including framework,
+  methods, input / output data schema, and requirements.
+
+![Iterative toolstack is modular](/uploads/images/2022-07-26/modular-toolstack.png)
+
+## UI of your choice
+
+Iterative Studio Model Registry can be used through your favorite interface,
+whether you’re a data scientist who likes APIs,a manager who prefers a web UI,
+or a DevOps engineer who works best with the command line interface (CLI).
+
```

Comment:
Should we be defining these roles connected to these interfaces this way?  It gives people the opportunity to say "that's not me" if we didn't match it up for them.  Instead, use the phrase that we are learned from PR folks that the "meet you where you are" messaging is a "powerful differentiator" according to Gartner folks.

So maybe?:
Iterative Model Studio Registry meets you where you are through your favorite interface.    Whether you like APIs, prefer a web interface or work best in the command line; whatever your role or preference, we've got you covered so your team can be most efficient.

Reply:
Sounds great. Changed. Thanks.